recipe/meta.yaml

```diff
@@ -8,75 +8,69 @@ package:
 source:
   - folder: dist
     url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: a0aeffe2fb1fdf374a8e4b471444f0f3ac4fb9f5a5b542b48824475e0042a5a6
+    sha256: e1875bb4b4e2de1669f4bc7869b6d3f54231cdced71605e6e64c9be77e3be50f
   - folder: src
     url: https://github.com/agronholm/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: 55a37ea9eb2092d435af148b42a10005ee62ec7eb7d7a12d73c88e2b8aca5e38
+    sha256: 64e2b1158ace6fee6c990ab78ee79707c0dea3f7f833a9541cd6ad056802ff87
 
 build:
   number: 0
-  # trigger: 0
-  script: cd dist && {{ PYTHON }} -m pip install . -vv --no-deps
-  skip: true  # [py<36]
+  script: cd dist && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<38]
 
 requirements:
-  build:
-    - m2-patch                               # [win]
-    - patch                                  # [not win]
   host:
     - pip
     - python
-    - setuptools-scm >=3.4
-    - setuptools >=42
+    - setuptools-scm >=6.4
+    - setuptools >=64
     - toml
-    - wheel >=0.29.0
+    - wheel
   run:
-    - contextvars  # [py<37]
-    - dataclasses  # [py<37]
+    - python
+    - exceptiongroup >=1.0.2  # [py<311]
     - idna >=2.8
-    - python >=3.6.2
     - sniffio >=1.1
-    - typing_extensions  # [py<38]
+    - typing_extensions >=4.1  # [py<311]
   run_constrained:
-    - trio >=0.16
+    - trio >=0.23
+    - uvloop >=0.17
 
 test:
   source_files:
     - src/tests
   requires:
-    - contextlib2  # [py<37]
-    - coverage >=4.5
-    - trio
+    - coverage >=7
+    - exceptiongroup >=1.2.0
     - hypothesis >=4.0
     - mock >=4
     - pip
-    - pytest >=6.0
+    - psutil >=5.9
+    - pytest >=7.0
     - pytest-cov
     - pytest-mock >=3.6.1
-    - toml
+    - tomli
+    - trio
     - trustme
     # won't be available on pypy, or windows, any time soon
-    - uvloop  # [unix and python_impl == 'cpython']
-    - ruamel_yaml  # [win]
+    - uvloop >=0.17  # [unix and python_impl == 'cpython']
   imports:
     - anyio
   commands:
-    - pip check || 1     # [win]
-    - pip check || true  # [unix]
-  # To check if downstream packages run without errors
-  downstreams:
-    # Testing jupyter_server sometimes fails in win platforms
-    - jupyter_server  # [unix]
+    - pip check
```

Comment:
The tests could be simplified by removing run_tests.py and including pyproject.toml for pytest configuration.
https://github.com/agronholm/anyio/blob/4.2.0/pyproject.toml#L92
```
test:
  source_files:
    - tests
    - pyproject.toml
  requires:
    - exceptiongroup >=1.2.0
    - hypothesis >=4.0.
    - pip
    - psutil >=5.9
    - pytest >=7.0.
    - pytest-mock >=3.6.1
    - toml
    - trio
    - trustme
    - uvloop >=0.17  # [unix]
  imports:
    - anyio
  commands:
    - pip check
    - pytest -m "not network"
```